backend/src/libs/packages/database/generated-schema/0000_pale_romulus.sql

```diff
@@ -0,0 +1,32 @@
+CREATE TABLE IF NOT EXISTS
+  "business_details" (
+    "id" serial PRIMARY KEY NOT NULL,
+    "company_name" varchar NOT NULL,
+    "tax_number" varchar NOT NULL,
+    "owner_id" integer NOT NULL,
+    "created_at" timestamp DEFAULT now() NOT NULL,
+    "updated_at" timestamp DEFAULT now() NOT NULL,
+    CONSTRAINT "business_details_company_name_unique" UNIQUE ("company_name"),
+    CONSTRAINT "business_details_tax_number_unique" UNIQUE ("tax_number")
+  );
+
+--> statement-breakpoint
+CREATE TABLE IF NOT EXISTS
+  "users" (
+    "id" serial PRIMARY KEY NOT NULL,
+    "phone" varchar NOT NULL,
+    "password_hash" varchar NOT NULL,
+    "password_salt" varchar NOT NULL,
+    "created_at" timestamp DEFAULT now() NOT NULL,
+    "updated_at" timestamp DEFAULT now() NOT NULL
+  );
```

Comment:
I guess something went wrong with migrations during merge conflicts. I believe we should take generated-schema dir from development and then run migrate:generate to keep everything consistent.

Reply:
There was an issue with 0001_powerful_sphinx.sql migration file. Name of business group has been inserted as 'Bu**is**ness', so I corrected it to 'Bu**si**ness'.